tests/functional_tests/cpp_backend_tests/test_driver.py

```diff
@@ -0,0 +1,58 @@
+import os
+import subprocess
+from pathlib import Path
+
+import pytest
+
+
+def _source_dir():
+    return Path(__file__).resolve().parent
+
+
+def _build_dir(backend_dir: str):
+    return _source_dir().joinpath(f"build_{backend_dir}")
```

Comment:
Just in case you don’t know, there’s also an operator version, e.g.:
```suggestion
    return _source_dir() / f"build_{backend_dir}"
```
But I don’t care if you use it.